Source/Common/md5.cpp

```diff
@@ -107,15 +107,15 @@ void MD5::update(const uint1 *input, uint4 input_length)
 
 void MD5::update(FILE *file)
 {
-    unsigned char buffer[1024];
+    unsigned char localBuffer[1024];

```

Comment:
this should stay as buffer, the member variable should be starting with m_ better to change that one then the local one